PWGGA/GammaConv/AliAnalysisTRDEfficiency.h

```diff
@@ -2,90 +2,50 @@
 /* See cxx source for full Copyright notice */
 /* $Id$ */
 
-#ifndef AliAnalysisTRDEfficiency_H
-#define AliAnalysisTRDEfficiency_H
+#ifndef AliAnalysisESDsTask_H
```

Comment:
I think this should stay as before

Reply:
Sorry, I didn't notice that this had changed